vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -52,66 +53,181 @@ void ReactImage::OnLoadEnd(winrt::event_token const &token) noexcept {
   m_onLoadEndEvent.remove(token);
 }
 
-winrt::fire_and_forget ReactImage::Source(ImageSource source) {
+void ReactImage::ResizeMode(react::uwp::ResizeMode value) {
+  if (m_resizeMode != value) {
+    m_resizeMode = value;
+
+    bool shouldUseCompositionBrush{m_resizeMode == ResizeMode::Repeat};
+    bool switchBrushes{m_useCompositionBrush != shouldUseCompositionBrush};
+
+    if (switchBrushes) {
+      m_useCompositionBrush = shouldUseCompositionBrush;
+      SetBackground(m_memoryStream != nullptr, false);
+    } else {
+      const auto bitmapBrush{Background().as<winrt::ImageBrush>()};
+      bitmapBrush.Stretch(ResizeModeToStretch(m_resizeMode));
```

Comment:
testing out these changes I'm getting a crash on this line with this callstack
![image](https://user-images.githubusercontent.com/48095233/70568974-de0c4800-1b4d-11ea-9c85-a3ad7dae5f70.png)

looks like the bitmapBrush is null while setting the Stretch

Reply:
In the Js i'm setting resizeMode={"cover"} on the <Image /> element

Reply:
I do want to share that although there is an issue when setting resizeMode with bitmap image. This does massively help with memory consumption when correctly sized images are not available (or dev forgets to resize) intentionally not resizing in my app this change get the total commit to 123MB down from 868MB